src/components/ItemDetail/index.jsx

```diff
@@ -0,0 +1,66 @@
+import getProductById from "../../api/getItemById";
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import styles from "./styles/DetailContent.module.css";
+import ItemDescription from "./ItemDescription";
+import AskSection from "./AskSection";
+import getItemComment from "../../api/getItemComment";
+import Review from "./component/Review";
+
+function DetailContent() {
+  const { productId } = useParams();
+  const [items, setItems] = useState(null);
+  const [comments, setComments] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const [itemData, commentData] = await Promise.all([
+          getProductById(productId),
+          getItemComment({ id: productId }),
+        ]);
+        setItems(itemData);
+        setComments(commentData.list);
+      } catch (error) {
+        console.error("데이터 불러오기 에러:", error);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+  }, [productId]);
+
+  if (loading || !items) {
+    return <p>상품 정보를 불러오는 중입니다...</p>;
+  }
+
+  console.log(comments);
+
+  return (
+    <main className={styles.main}>
+      <ItemDescription items={items} />
+      <AskSection />
+      <section className={styles.review}>
+        {comments.length > 0 &&
+          comments.map((comment, index) => (
+            <Review key={index} comment={comment} />
+          ))}
```

Comment:
❗️ 수정요청
key는 렌더링 최적화를 위한 값으로 key로 사용할 수 있는 다른 유니크한 값이 있다면 해당 값을 이용해주세요~
저라면 comment id 값을 이용할 것 같아요!

```suggestion
          comments.map((comment) => (
            <Review key={comment.id} comment={comment} />
          ))}
```
https://panda-market-api.vercel.app/docs/#/Comment/ListProductComments